### ISSUE TITLE
Create a script to update the version of Octicons used

### ISSUE THREAD

**Issue by lee-dohm** [lee-dohm/octicons-ex#28]
The steps consist of:

```
# Update `package.json`
npm update
cp node_modules/octicons/build/data.json priv/data.json
cp node_modules/octicons/package.json priv/package.json
```

Would also be nice to have a system for extracting the Octicons version number from `package.json` to include in the documentation for `Octicons.version()`.

**Comment by lee-dohm** [comment]
If we decide to go with the "pre-convert" alternative in #24, then this script should handle that step too.

**Comment by lee-dohm** [comment]
Fixed in #30
